kong/plugins/grpc-web/handler.lua

```diff
@@ -0,0 +1,91 @@
+-- Copyright (c) Kong Inc. 2020
+
+local deco = require "kong.plugins.grpc-web.deco"
+
+local ngx = ngx
+local kong = kong
+
+local string_format = string.format
+local ngx_req_get_method = ngx.req.get_method
+local kong_request_get_path = kong.request.get_path
+local kong_request_get_header = kong.request.get_header
+local kong_request_get_raw_body = kong.request.get_raw_body
+local kong_response_exit = kong.response.exit
+local kong_response_set_header = kong.response.set_header
+local kong_service_request_set_header = kong.service.request.set_header
+local kong_service_request_set_raw_body = kong.service.request.set_raw_body
+
+
+local grpc_web = {
+  PRIORITY = 1,
```

Comment:
Maybe use `2` or `3`, as `1` conflicts with `correlation-id`. `2` is used by `request-termination` which does similar things (generating a response).

https://github.com/Kong/kong/blob/next/kong/plugins/request-termination/handler.lua#L23

Reply:
done, thanks